aruba.gemspec

```diff
@@ -46,7 +46,8 @@ Gem::Specification.new do |spec|
   spec.rubygems_version = '>= 1.6.1'
   spec.required_ruby_version = '>= 2.4'
 
-  spec.files = Dir['{docs,fixtures,lib}/**/*', 'CHANGELOG.md', 'LICENSE', 'README.md']
+  spec.files = Rake::FileList['{docs,fixtures,lib}/**/*', 'CHANGELOG.md', \
```

Comment:
This PR now does not have the explicit `require 'rake/file_list'` - is that significant?

Reply:
Eeks. Thanks, @olleolleolle. Fixed! :heart: